src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/ExportIPOs/ExportIpoRepository.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Transactions;
+using Dapper;
+using Equinor.ProCoSys.Common.Misc;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Repositories.ExportIPOs;
+
+
+public class ExportIpoRepository : DapperRepositoryBase, IExportIpoRepository
```

Comment:
consider adding unit tests, if possible. I see that OutstandingIpoRepository has some tests, building a context using sqllite. Maybe create a PBI for next sprint and leave a todo here with // Todo <PBI-number> Create unit tests

Reply:
Here is a PBI for this: https://dev.azure.com/statoildeveloper/Statoil.NextGenProCoSys/_workitems/edit/109937

Reply:
Add a comment above the method: // Todo 109937 Add unit tests